src/layout/grid/grid.jsx

```diff
@@ -330,6 +338,24 @@ const Grid = React.forwardRef(
             },
         );
 
+        if (responsiveQueryType === 'container') {
+            console.log('pre gridColumns', children);
+        }
+        const gridColumns = responsiveQueryType === 'container' ?
+            children.map( // <GridColumn />'s with @container queries enabled
+                (child) => ({
+                    ...child,
+                    props: {
+                        ...child.props,
+                        responsiveQueryType: 'container',
+                    },
+                }),
+            ) :
+            children;
+
+        if (responsiveQueryType === 'container') {
+            console.log('gridColumns', gridColumns);
+        }
```

Comment:
Same here: please remove when you're done debugging

Reply:
Removed!